spec/fixtures/comments.yml

```diff
@@ -65,8 +65,8 @@ public_comment_on_asset_by_user:
   commentable: valid_mp3 (Asset)
   body: "Well friend, this is your best work."
   is_spam: false
-  user: sudara
-  commenter: arthur
+  user: arthur
+  commenter: sudara
```

Comment:
These changes are why Percy is ❌ — If you go to the build https://percy.io/alonetone/alonetone/builds/1628431 and look at the originals (not the diff) you can see that the home page comment disappeared. This is probably due to the "don't show comment on track from same user/ip" logic we added. We need to comment from the fixtures showing in Percy, so maybe better to add fixtures for your particular use if you need the owner switched here!

Reply:
Ah I didn't realize it was using spec files to fill those pages. I'd expect something like seed.rb or something. Good to know.